magento2.go

```diff
@@ -44,6 +63,140 @@ func (m2 *magento2) ParseConfig(cfgPath string) (*StoreConfig, error) {
 	}, nil
 }
 
+func (m2 *magento2) BaseURLs(docroot string) ([]string, error) {
+	cfgPath := fmt.Sprintf("%s/%s", docroot, m2.ConfigPath())
+
+	urls, err := m2.getBaseURLsFromDatabase(cfgPath)
+	if err == nil {
```

Comment:
Now any DB error is lost. If you were to merge the results (from DB & from config), perhaps also merge any errors?